src/neuroconv/datainterfaces/behavior/lightningpose/lightningposeconverter.py

```diff
@@ -70,20 +70,11 @@ def __init__(
             self.data_interface_objects.update(dict(LabeledVideo=VideoInterface(file_paths=[labeled_video_file_path])))
 
     def get_conversion_options_schema(self) -> dict:
-        conversion_options = self.data_interface_objects["PoseEstimation"].get_conversion_options_schema()
-        conversion_options = dict_deep_update(
-            conversion_options, self.data_interface_objects["OriginalVideo"].get_conversion_options_schema()
+        conversion_options_schema = get_schema_from_method_signature(
+            method=self.add_to_nwbfile, exclude=["nwbfile", "metadata"]
```

Comment:
I thought that your config would allow those ones to work by just checking instances.

Reply:
Good question - the original reasoning is because that's how it is now: https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/basedatainterface.py#L47

But taking a closer look on this PR...

The issue is that `pynwb.NWBFile` has no valid JSON type - setting `arbitrary_types_allowed=True` in the config allows any field of any type in the Pydantic model to be set, but then JSON schema generation fails for such a model without custom decoding, and I'm not really sure how I would describe that in-memory object in JSON for that matter

As such, the `nwbfile` argument really should be skipped

But I will see about relaxing the `metadata` restriction - previously that would have been because we couldn't support anything too complicated, like `Dict[str, Any]` but maybe now we can

Reply:
Yes. When I wrote this I did not relize that under the hood we will still be using the json schema to do the validation.

passing the config like this allows you to do pydantic validation by instance but leaves you with no json schema.